urllib3/poolmanager.py

```diff
@@ -110,8 +119,14 @@ class PoolManager(RequestMethods):
 
     proxy = None
 
-    def __init__(self, num_pools=10, headers=None, **connection_pool_kw):
+    def __init__(self, num_pools=10, headers=None, transport_security_manager=_DEFAULT_TSM,
+                 **connection_pool_kw):
         RequestMethods.__init__(self, headers)
+
+        if transport_security_manager is _DEFAULT_TSM:
+            transport_security_manager = TransportSecurityManager()
```

Comment:
Sorry, I'm not sure if I follow. If I call `PoolManager()`, `_DEFAULT_TSM` gets overwritten with an actual TSM. If I call `PoolManager(transport_security_manager=None)`, then `None` gets passed down the stack, disabling TSM. Any other value will also get passed down the stack and result in an error if it's not a TSM. Am I missing anything?